workers/unity/Assets/Gdk/PlayerLifecycle/Tests/Editmode/HandleCreatePlayerRequestSystemTests.cs

```diff
@@ -0,0 +1,76 @@
+using System.Linq;
+using Generated.Improbable.PlayerLifecycle;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.PlayerLifecycle;
+using Improbable.Gdk.TestUtils;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+[TestFixture]
+public class HandleCreatePlayerRequestSystemTests : HybridGdkSystemTestBase
+{
+    private UnityTestWorker worker;
+
+    [SetUp]
+    public void Setup()
+    {
+        worker = new UnityTestWorker("worker-id", new Vector3());
+    }
+
+    [TearDown]
+    public void TearDown()
+    {
+        worker.Dispose();
+    }
+
+    [Test]
+    public void
```

Comment:
Thinking about making this test more useful, right now it does a relatively meaningless assertion